boards/FSM/Src/Io/Io_Coolant.c

```diff
@@ -0,0 +1,84 @@
+#include <assert.h>
+#include <math.h>
+#include "main.h"
+#include "Io_Coolant.h"
+#include "Io_SharedFreqOnlyPwmInput.h"
+
+//TODO set these values
+#define TEMPERATURE_VOLTAGE_MIN (0.0f)
+#define TEMPERATURE_VOLTAGE_MAX (1.0f)
+#define PRESSURE_VOLTAGE_MIN (0.0f)
+#define PRESSURE_VOLTAGE_MAX (1.0f)
+
+static struct FreqOnlyPwmInput *flow_meter;
+
+void  Io_FlowMeter_Init(TIM_HandleTypeDef *htim)
+{
+    assert(htim != NULL);
+
+    flow_meter = Io_SharedFreqOnlyPwmInput_Create(
+        htim, TIMx_FREQUENCY / TIM4_PRESCALER, TIM_CHANNEL_2, TIM4_AUTO_RELOAD_REG, HAL_TIM_ACTIVE_CHANNEL_2);
+}
+float Io_FlowMeters_GetFlowRate(void)
+{
+    return Io_SharedFreqOnlyPwmInput_GetFrequency(flow_meter) / 7.5f;
```

Comment:
Can #define this as a constant

Reply:
this is function? I also don't know what 7.5 does so I wouldn't know what to call it

Reply:
![image](https://user-images.githubusercontent.com/28274296/201489431-4ce2f7ef-32f1-4071-a14b-d89c88db523f.png)
From the coolant flow sensor (https://www.adafruit.com/product/828)
The P/N is on the schematic too

Reply:
fixed 579f9c1